src/Package.php

```diff
@@ -65,6 +65,23 @@ public static function get_path() {
 		return self::get_package()->get_path();
 	}
 
+	/**
+	 * Checks if we're executing the code in an experimental build mode.
+	 *
+	 * @return boolean
+	 */
+	public static function is_experimental_build() {
+		return WOOCOMMERCE_BLOCKS_PHASE > 2;
+	}
+
+	/**
+	 * Checks if we're executing the code in an feature plugin or experimental build mode.
+	 *
+	 * @return boolean
+	 */
+	public static function is_feature_plugin_build() {
+		return WOOCOMMERCE_BLOCKS_PHASE > 1;
+	}
```

Comment:
Please add the code in `src/Domain/Package.php` and then this file will just wrap the calls to the methods there (similar to other methods in this class).

It may seem odd but there's a transitory refactor in process that's not complete yet. `src/Package.php` is required because of the current way things are implemented in WooCommerce core (and that might never change), but we're transitioning to using DI in the blocks code (code debt!) where `src/Package` will eventually _only_ be used for the package registration (and kept mostly for back-compat) in WooCommerce core and `src/Domain/Package` will be injected wherever it's methods are needed.